accessibility-checker-engine/help/HAAC_Video_HasNoTrack.mdx

```diff
@@ -40,8 +41,8 @@ For example:
 
 ### About this requirement
 
-[IBM 1.2.1 Audio-only and Video-only (Prerecorded)](http://www.ibm.com/able/guidelines/ci162/audio_and_video.html)
-[WCAG 2.1 technique H96](https://www.w3.org/WAI/WCAG21/Techniques/html/H96)
+[IBM 1.2.2 Captions (Prerecorded)](https://www.ibm.com/able/guidelines/ci162/captions.html)
+[WCAG 2.1 technique H95](https://www.w3.org/WAI/WCAG21/Techniques/html/H95)
 
 ### Who does this affect?
 
```

Comment:
Now that we have removed audio description, we also need to remove 'Blind people using screen readers' from the 'Who does this affect' section.

Reply:
Done!